ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminViewsDcatRestController.java

```diff
@@ -0,0 +1,88 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.admin.rest.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.apache.jena.rdf.model.Model;
+
+import static org.apache.jena.riot.WebContent.contentTypeJSONLD;
+import static org.apache.jena.riot.WebContent.contentTypeNQuads;
+import static org.apache.jena.riot.WebContent.contentTypeTurtle;
+
+@Tag(name = "Views DCAT")
+@SuppressWarnings("java:S2479") // whitespace needed for examples
+public interface AdminViewsDcatRestController {
```

Comment:
Would it be an idea of Calling these "OpenApiController" and the implementation "RestController" ? 
Since seeing "Impl" on top of a Controller is a bit strange